### PR TITLE
[SW-258] Move components and pages from popup/router/ dir to popup/ (one level up)

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/components/Modals/ShareQr.vue [hunk 1]
```diff
@@ -81,8 +81,8 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@use "../../../../styles/share-info";
-@use "../../../../styles/variables";
+@use "../../../styles/share-info";
+@use "../../../styles/variables";
```
Comment (pstachel) [inline src/popup/components/Modals/ShareQr.vue:85]: Singlequotes should be used

File: src/popup/components/Modals/RecipientInfo.vue [hunk 2]
```diff
@@ -88,10 +88,10 @@ export default {
...
 
 <style lang="scss" scoped>
-@use "../../../../styles/share-info";
-@use "../../../../styles/variables";
-@use '../../../../styles/typography';
-@use '../../../../styles/mixins';
+@use "../../../styles/share-info";
+@use "../../../styles/variables";
+@use '../../../styles/typography';
+@use '../../../styles/mixins';
```
Comment (pstachel) [inline src/popup/components/Modals/RecipientInfo.vue:94]: Single quotes should be used

File: src/popup/components/Modals/RecipientHelper.vue [hunk 3]
```diff
@@ -57,9 +57,9 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@use "../../../../styles/share-info";
-@use "../../../../styles/variables";
-@use '../../../../styles/typography';
+@use "../../../styles/share-info";
+@use "../../../styles/variables";
+@use '../../../styles/typography';
```
Comment (pstachel) [inline src/popup/components/Modals/RecipientHelper.vue:62]: Single quotes should be used

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/popup/components/Modals/RecipientHelper.vue (hunk 3 above)
Comment (peronczyk) [reply src/popup/components/Modals/RecipientHelper.vue]: Fixed

File: src/popup/components/Modals/ShareQr.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/Modals/ShareQr.vue]: Fixed

**Review by pstachel — APPROVED** [review]

**Review by ifaouibadi — APPROVED** [review]

**Pull request merged**